openmp/device/include/DeviceTypes.h

```diff
@@ -21,6 +21,8 @@ template <typename T> using Constant = __gpu_constant T;
 template <typename T> using Local = __gpu_local T;
 template <typename T> using Global = __gpu_local T;
 
+#define omp_invalid_device -2
```

Comment:
Please add a comment here that connects this line to `omp.h.var` and `omp.f90.var` to make sure what if it is changed, also the external visible OpenMP interface needs to change.